Library/ViewModels/ChangeEmailViewModel.swift

```diff
@@ -46,11 +46,17 @@ public protocol ChangeEmailViewModelType {
 public final class ChangeEmailViewModel: ChangeEmailViewModelType, ChangeEmailViewModelInputs,
 ChangeEmailViewModelOutputs {
   public init() {
+    self.dismissKeyboard = Signal.merge(
```

Comment:
Issue w/ this is that it will update the email even if the save button is disabled. The save button is disabled when the user enters their current email address which prevents them from "updating" their email w/ their current email. The user can still "update" their email through the return key. currently working on fixing this bug but would be happy to get feedback on this.